app/scripts/services/dimStoreService.factory.js

```diff
@@ -1593,5 +1610,35 @@
       return ret;
     }
     // code above is from https://github.com/DestinyTrialsReport
+
+    // code below is for declassifying items
+    function loadJSON(file) {
+      return $http.get(file)
+        .then(function(json) {
+          return json.data.itemHash;
+        });
+    }
+
+    function declassify(itemDef, language, classifiedData) {
+      classifiedData = classifiedData.$$state.value;
```

Comment:
If anyone has a better idea on how to do this once instead of each time, that would be great. I'm probably over-looking something obvious.